toolversions.sh

```diff
@@ -42,6 +42,7 @@ install_docfx() {
   if [[ ! -f $DOCFX ]]
   then
     (echo "Fetching docfx v${DOCFX_VERSION}";
+     mkdir -p $REPO_ROOT/packages;
```

Comment:
I'm not really sure why this, this just pulls in the zip but AFAICT will fix the current issue.

@jskeet any reason to not use nuget?